ts/store/reducers/backendStatus.ts

```diff
@@ -210,6 +210,20 @@ export const bancomatPayConfigSelector = createSelector(
     )
 );
 
+/**
+ * return the remote config about LolliPOP enabled/disabled
+ * if there is no data, false is the default value -> (LolliPOP disabled)
+ */
+export const isLollipopEnabledSelector = createSelector(
+  backendStatusSelector,
+  (backendStatus): boolean =>
+    pipe(
+      backendStatus,
+      O.map(bs => bs.config.lollipop.enabled),
+      O.toUndefined
+    ) ?? false
+);
```

Comment:
It's not an error but just curiosity why not use `O.gerOrElseW` instead `O.toUndefined` removing nullish op

Reply:
Good point! Just because I copied from an existing one tbh :)
I would say that here should be enough `O.getOrElse` because the type returned is the same:

```ts
export const isLollipopEnabledSelector = createSelector(
  backendStatusSelector,
  (backendStatus): boolean =>
    pipe(
      backendStatus,
      O.map(bs => bs.config.lollipop.enabled),
      O.getOrElse(() => false)
    )
);
```